src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -465,6 +469,8 @@ public function getContent()
      * @param string $version The HTTP protocol version
      *
      * @return $this
+     *
+     * @final since version 3.2
```

Comment:
`@final` support is since 3.3, but the method is final since 3.2 - see deprecation in the constructor